board/rt1051/crashdump/crashdumpwriter_vfs.hpp

```diff
@@ -15,21 +16,21 @@ namespace purefs::fs
 
 namespace crashdump
 {
-    constexpr inline auto maxFilesCount = 5;
+    constexpr inline auto maxRotationFilesCount = 5;
     class CrashDumpWriterVFS
     {
       public:
-        CrashDumpWriterVFS()
+        CrashDumpWriterVFS() : rotator{".hex"}
```

Comment:
I believe logs should have .log or .txt extension (rotator c-tor has variable with `log` in name ;) )

Reply:
But this is for crashdumps and here the extension is/was .hex :)

Reply:
But in this case, it could be an empty string as well

Reply:
I meant - if rotator is extension agnostic, then it should not have fields with log in name, but that's minor :P